src/aiq/runtime/session_metadata.py

```diff
@@ -16,18 +16,18 @@
 from starlette.datastructures import Headers
 from starlette.datastructures import QueryParams
 
-from aiq.data_models.api_server import Request
+from aiq.data_models.api_server import RequestAttributes
 
 
-class RequestAttributes:
+class SessionMetadata:
     """
-    The RequestAttributes class is responsible for managing user-defined
-    metadata and attributes. It provides a way to store and
-    expose user-defined attributes to workflow tools.
+    The SessionMetadata class is responsible for managing user http and webscoket session
+    metadata. It provides a way to store and expose session attributes to workflow tools.
     """
 
     def __init__(self) -> None:
-        self._request: Request = Request()
+        self._request: RequestAttributes = RequestAttributes()
+        self._thread_id: str | None = None
```

Comment:
The thread_id should be a property of the context, not a property of the request.